dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2014, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The Contexts and Dependency Injection 3.0 specification makes it easier to integrate \
```

Comment:
Change,
"The Contexts and Dependency Injection 3.0 specification makes it easier to integrate
 Jakarta EE components of different types. It provides a common mechanism to inject component such as EJBs or Managed Beans into other components such as JSPs or other EJBs."
to,
"With the Contexts and Dependency Injection 3.0 specification, you can integrate Jakarta EE components of different types. It provides a common mechanism to inject component such as EJBs or Managed beans into other components such as JSPs or other EJBs."